lib/twitter_cldr/version.rb

```diff
@@ -4,5 +4,5 @@
 # http://www.apache.org/licenses/LICENSE-2.0
 
 module TwitterCldr
-  VERSION = "3.1.2"
+  VERSION = "3.2.0"
```

Comment:
Probably only a minor version bump required for this PR since your changes are purely additive.